### ISSUE TITLE
@bugsnag/react-native - v7.17.1 error when running pod install (iOS)

### ISSUE THREAD

**Issue by stawecki** [bugsnag/bugsnag-js#1778]
### Describe the bug
Installing CocoaPods on a React Native project fails when using package version 7.17.1

It seems the release is missing files in directory `packages/react-native/ios/vendor/bugsnag-cocoa`.

### Steps to reproduce
1. Go to a standard React Native project.
2. `yarn add @bugsnag/react-native@7.17.1`
3. `cd ios && pod install`

### Environment
* Bugsnag version: 7.17.1

### Workaround
* Downgrade to previous version
    * With yarn `yarn add @bugsnag/react-native@7.17.0`
    * Or update package.json to use `"7.17.0"` and reinstall modules

<details><summary>Error messages:</summary>

```
[!] Invalid `Podfile` file:
[!] Invalid `BugsnagReactNative.podspec` file: No such file or directory @ rb_sysopen - /Users/xxx/Projects/xxx/node_modules/@bugsnag/react-native/ios/vendor/bugsnag-cocoa/Bugsnag.podspec.json.

 #  from /Users/xxx/Projects/xxx/node_modules/@bugsnag/react-native/BugsnagReactNative.podspec:3
 #  -------------------------------------------
 #  package = JSON.parse(File.read(File.join(__dir__, "package.json")))
 >  bugsnag_cocoa_podspec = JSON.parse(File.read(File.join(__dir__, "ios", "vendor", "bugsnag-cocoa", "Bugsnag.podspec.json")))
 #  bugsnag_cocoa_public_header_files = bugsnag_cocoa_podspec["public_header_files"]
 #  -------------------------------------------
.

 #  from /Users/xxx/Projects/xxx/ios/Podfile:20
 #  -------------------------------------------
 #    end
 >    config = use_native_modules!
 #    use_react_native!(:path => config["reactNativePath"])
```
</details>

**Comment by luke-belton** [comment]
Hi @stawecki - thanks for reporting this, We're investigating and will keep you updated on this thread.

**Comment by dylanphenix** [comment]
Same here !

**Comment by swrobel** [comment]
FWIW 7.17.0 doesn't have any issue

**Comment by stawecki** [comment]
Fix confirmed in 7.17.2

Thanks many 🙇

**Comment by luke-belton** [comment]
Hi all - yes just to confirm a fix for this was released earlier in [`v7.17.2`](https://github.com/bugsnag/bugsnag-js/releases/tag/v7.17.2) 🎉  Thanks for reporting!
